threatbus/data.py

```diff
@@ -112,3 +115,180 @@ class Sighting:
     ts: datetime
     intel: str
     context: dict
+
+
+@dataclass()
+class SnapshotEnvelope:
+    snapshot_type: MessageType
+    snapshot_id: str
+    body: Intel or Sighting
+
+
+@dataclass
+class SnapshotRequest:
+    snapshot_type: MessageType
+    snapshot_id: str
+    snapshot: timedelta
```

Comment:
Could we have some documentation for these data classes?

Reply:
I added explanatory comments for the two data classes.